csrc/lower_validation.cpp

```diff
@@ -634,13 +673,8 @@ void fillProducerVectorizedContigRootDomains(
 
   VectorizedSetInfo& info = *it;
 
-  // info.vectorized_root_id is validated at this point to be the
-  // last concrete root domain in consumer.
-  auto consumer_root_id = info.vectorized_root_id;
-
-  auto root_id = c2p_map.at(consumer_root_id);
-
-  fillVectorizedContigRootDomains(producer_tv, contig_finder, root_id, info);
+  fillVectorizedContigRootDomains(
```

Comment:
`c2p_map` is no longer used, so can be removed.

Reply:
removed